app/controllers/status_messages_controller.rb

```diff
@@ -49,6 +49,8 @@ def create
       format.mobile { redirect_to stream_path }
       format.json { render json: PostPresenter.new(status_message, current_user), status: 201 }
     end
+  rescue StatusMessageCreationService::BadAspectsIDs
+    render status: 422, plain: "Provided aspects IDs aren't applicable (non-existent or not owned)"
```

Comment:
Even if this should probably never happen, but when somebody starts writing a post in one tab and then deletes an aspect in another tab, this could still be a normal scenario (so not trying to modify the request). So I think this error-message should be added to the translations.